### PR TITLE
Added javascript injector for logging into Dynatrace

### PR DESCRIPTION
**Pull request by iamlawrenceonoja**

### CODE REVIEW

File: javascript-injectors/examples/dynatrace-login-via-credentials.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+function dynatraceLogin () {
+  const email = 'oabah@screenly.io'
+  const password = 'Abcdefghijkl$'
```
Comment (iamplugged) [inline javascript-injectors/examples/dynatrace-login-via-credentials.js:3]: @iamlawrenceonoja Replace your credentials with placeholders

**Review by iamplugged — CHANGES_REQUESTED** [review]
Minor comment

**Comment by iamlawrenceonoja** [comment]
> Minor comment

Thanks. I've made changes

**Comment by iamlawrenceonoja** [comment]
@iamplugged You can approve now. Thanks

**Review by iamplugged — APPROVED** [review]

**Pull request merged**